Dockerfile

```diff
@@ -35,5 +38,7 @@ USER mumble
 # Set volumes
 VOLUME /etc/mumble
 
-# Default command
-CMD ["/opt/mumble/murmur.x86", "-fg", "-ini", "/etc/mumble/config.ini"]
+# Entrypoint
+COPY entrypoint.sh /app/entrypoint.sh
+ENTRYPOINT [ "/app/entrypoint.sh" ]
+CMD [ "/opt/mumble/murmur.x86", "-fg", "-ini" , "${CONFIG_PATH}" ]
```

Comment:
The "exec" form of `CMD` doesn't do variable substitution, therefor `${CONFIG_PATH}` doesn't work here. We should use the "shell" form instead.

```suggestion
CMD /opt/mumble/murmur.x86 -fg -ini ${CONFIG_PATH}
```

From the [Dockerfile reference](https://docs.docker.com/engine/reference/builder/) docs:
![image](https://user-images.githubusercontent.com/53531/77781216-68e85400-7012-11ea-80e5-46307c70bc84.png)

NOTE: Moving this to the shell script as noted above would also resolve this.

Reply:
From my understanding of your suggestions it seems like they do not all work towards the same solution. Now I'm a bit confused as to which suggestion I should use and which not.

Reply:
Sorry, that was supposed to be `CMD /app/entrypoint.sh`